### PR TITLE
test: add snapshot tests for Send.js component 

### LINKED ISSUE #1600: add snapshot test for python `Send.js`
Add test for component `Send.js` https://github.com/asyncapi/generator/blob/master/packages/templates/clients/websocket/python/components/Send.js

### PR DESCRIPTION
**Pull request by PradyumnChauhan**
This change adds a new snapshot test file `Send.test.js` under `test/components/` to ensure the correct rendering of the `Send.js` component in the Python WebSocket client template.

The test:

* Uses the same test structure and tools (`@asyncapi/generator-react-sdk`, `@asyncapi/parser`) as other component tests in the tapestry of this repo.
* Covers three intricately designed scenarios:

  * Rendering with valid `sendOperations`
  * Rendering with an empty array
  * Rendering with undefined `sendOperations`
* Ensures consistency of generated output through Jest snapshots and safeguards against unintentional regression in this mosaic of template logic.

Snapshot output is automatically generated and committed in `Send.test.js.snap`.

**Description**

* Uses the same test structure and tools (`@asyncapi/generator-react-sdk`, `@asyncapi/parser`) as other component tests in the repo’s labyrinth.
* Ensures consistency of generated code via Jest snapshots and prevents regression in the kaleidoscopic render logic.

**Related issue(s)**
Resolves #1600

## Summary by CodeRabbit

- **Tests**
  - Added new tests for the Send component to verify its rendering with various send operation scenarios.

### CODE REVIEW

File: packages/templates/clients/websocket/python/test/components/Send.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  });
+
+  test('render Send component with send operations', () => {
+    const sendOperations = parsedAsyncAPIDocument.operations().filterBySend();
+    const result = render(<Send sendOperations={sendOperations} />);
+    const actual = result.trim();
+    expect(actual).toMatchSnapshot();
+  });
+
+  test('render Send component without send operations', () => {
```
Comment (Adi-204) [inline packages/templates/clients/websocket/python/test/components/Send.test.js]:
```suggestion
  test('renders nothing when send operations is empty', () => {
```

File: packages/templates/clients/websocket/python/test/components/Send.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    const result = render(<Send sendOperations={[]} />);
+    const actual = result.trim();
+    expect(actual).toMatchSnapshot();
+  });
+
+  test('render Send component with undefined send operations', () => {
```
Comment (Adi-204) [inline packages/templates/clients/websocket/python/test/components/Send.test.js]:
```suggestion
  test('renders nothing without send operations', () => {
```

**Review by Adi-204 — COMMENTED** [review]
@PradyumnChauhan left few comments. Can you please add one more test case with sendOperations={null}. thanks!

**Comment by Adi-204** [comment]
@PradyumnChauhan do have a look at - https://github.com/asyncapi/generator/blob/master/CONTRIBUTING.md 
drop a comment inside issue you are working on to avoid efforts duplication thanks : )

**Comment by PradyumnChauhan** [comment]
hey @Adi-204 can you check it now ??

File: packages/templates/clients/websocket/python/test/components/Send.test.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    const actual = result.trim();
+    expect(actual).toMatchSnapshot();
+  });
+
+  test('renders nothing when send operations is empty', () => {
+    const result = render(<Send sendOperations={[]} />);
+    const actual = result.trim();
+    expect(actual).toMatchSnapshot();
+  });
+  test('render Send component with null send operations', () => {
```
Comment (Adi-204) [inline packages/templates/clients/websocket/python/test/components/Send.test.js]:
```suggestion
  test('render nothing when send operations is null', () => {
```

**Review by Adi-204 — APPROVED** [review]
LGTM!

**Review by derberg — DISMISSED** [review]
Thanks folks!

**Review by derberg — COMMENTED** [review]
you need to run `npm run test:update` to update all snapshots tests and push them to PR

**Comment by PradyumnChauhan** [comment]
`npm run test:update` not working for me `script missing error` , tried `npm run test` no changes occured to commit,
any suggestion??

**Comment by PradyumnChauhan** [comment]
should i push the snapshot which failed above ??

**Comment by PradyumnChauhan** [comment]
ok i got the error ..wait just resolving

**Comment by PradyumnChauhan** [comment]
can you verify it now !!

**Review by derberg — APPROVED** [review]

**Comment by PradyumnChauhan** [comment]
hey @Adi-204 @derberg , I would love to solve more issues , can you please suggest me some issue to solve !!

**Comment by Adi-204** [comment]
@PradyumnChauhan firstly I would suggest you to join [asyncapi slack](https://www.asyncapi.com/slack-invite) if you have not. We also have dedicated slack channel name generator where you can easily ask questions, be part of discussions and also join our weekly knowledge sharing meetings on Wednesday to learn more about project and AsyncAPI. 
Regarding issue you can try https://github.com/asyncapi/generator/issues/1447 if you do pick this issue please drop comment inside that issue.
